src/current/_includes/v24.1/known-limitations/plpgsql-feature-limitations.md

```diff
@@ -1,10 +1,7 @@
-- PL/pgSQL blocks cannot be nested. [#114775](https://github.com/cockroachdb/cockroach/issues/114775)
 - PL/pgSQL arguments cannot be referenced with ordinals (e.g., `$1`, `$2`). [#114701](https://github.com/cockroachdb/cockroach/issues/114701)
 - `FOR` loops, including `FOR` cursor loops, `FOR` query loops, and `FOREACH` loops, are not supported. [#105246](https://github.com/cockroachdb/cockroach/issues/105246)
```

Comment:
These three are about statements that aren't yet supported. I wonder if it makes sense to combine them into one "The following statements are not supported:", with three items.